_pages/en_US/troubleshooting.txt

```diff
@@ -453,6 +453,24 @@ You did not copy the `gm9` folder from the GodMode9 `.zip` to the root of your S
 {% endcapture %}
 <details>{{ compat | markdownify }}</details>
 
+{% capture compat %}
+<summary><u>"Backup failed" or "Error: Could not open directory" when attempting a NAND backup</u></summary>
+Make sure you have at least 1.3GB available in your SD card. If you don't have enough space, follow the steps:
+1. Power off your console
+1. Remove the SD card and insert it into your computer
+1. Copy the `Nintendo 3DS` folder from the root of your SD card to your computer
+1. Delete it from the SD card
+1. Load GodMode9 by holding START
```

Comment:
Use "power off, hold start, power on still holding start" as seen in rest of guide